package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-scm-base",
-  "version": "2.4.0",
+  "version": "3.0.0",
```

Comment:
Yup that sounds right.

The difference is that this change does not affect or modify the previous API. Since the previous API continues to work as is, I'd consider this update as backwards compatible.